docs/RestrictedAPIExamples.rst

```diff
@@ -0,0 +1,121 @@
+===================================
+Python Restricted Endpoint Examples
+===================================
+
+The HCA API provides several ways for users of the Human Cell Atlas (HCA) to access and download
+data sets from the HCA. This page covers how to access the HCA using Python API bindings.
+
+The API calls listed here are restricted to those with upload or ingest permissions.
+Data will be submitted through a single Ingestion Service API. Submitted data will go through
+basic quality assurance before it is deposited into the Data Storage System (DSS) component.
+
+In the document that follows, *privileged user* refers to a user with proper credentials and
+permission to upload/ingest data into the DSS.
+
+*NOTE:* The HCA CLI utility is compatible with Python 3.5+.
+
+
+``delete_bundle``
+-----------------
+
+Deletes an existing bundle given a UUID, version, and replica.
+
+Example call to ``delete_bundle()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/delete_bundle_api.py
+
+``put_bundle``
+--------------
+
+Creates a bundle. A bundle can contain multiple files of arbitrary type.
+
+Inputs:
```

Comment:
`put_bundle` lists this glossary of inputs, but `delete_bundle` does not.  Is there a consistent and efficent way of managing this (should we link to an actual glossary page whenever these keywords are brought up)?

Reply:
Done in 4daef94